docs/distributions/gentoo/installation.md

```diff
@@ -0,0 +1,60 @@
+# Installing Gentoo on a Mac with the T2 Chip
+
+You will need:
+
+- USB drive with at least 1GB
+- A way to plug it into your Mac (USB-C isn't USB-A)
+- A wired internet connection (i.e. USB-C to Enternet dongle) or wifi (not all models support it currently), check [the guide](https://wiki.t2linux.org/guides/wifi/) for compatibility and instructions. Its also technically possible to perform an offline installation, see [this](https://wiki.archlinux.org/index.php/Pacman/Tips_and_tricks#Installing_packages_from_a_CD/DVD_or_USB_stick) (retrieve the packages from an Arch virtual machine or Docker container)
```

Comment:
The arch instructions do not really apply here as you are using Ubuntu for the install process